packages/excalidraw/element/flowchart.ts

```diff
@@ -416,8 +416,8 @@ const createBindingArrow = (
     type: "arrow",
     x: startX,
     y: startY,
-    startArrowhead: appState.currentItemStartArrowhead,
-    endArrowhead: appState.currentItemEndArrowhead,
+    startArrowhead: undefined,
+    endArrowhead: "arrow",
```

Comment:
should we make `endArrowhead` this respect the setting if it's arrow-like? (`arrow`/`triangle`/`triangle_outline`)

Reply:
I am sorry I didn't get your point,

But look if you want the arrowhead to follow what the current app State is following then you should not change the anything and this PR is not useful the original code is fine but if you want that Evey single time when you create a flowchart and a specific type of arrow then my PR is useful.
But my code is always going to create a right Arrow okey.

But if you like my suggestion I am going to suggest you to make some options for user to choose what kind of Arrow he wants in his flowchart that's going to be more useful.

And if you want some more info please let me know
And if this comment doesn't answer your question please let me know okey

Reply:
Yes, the #8581 PR does exactly this @dwelle.